docs/architecture/overview.md

```diff
@@ -64,6 +64,7 @@ This document specifies the architecture for GCP Ingestion as a whole.
   1. Validate `payload` using a JSON Schema determined by attributes
   1. Parse `agent` attribute into `user_agent_*` attributes
   1. Produce `normalized_` variants of select attributes
+  1. Allow sanitization of attributes, configurable per document type via schema metadata
```

Comment:
Why is this `Allow sanitization` instead of `Sanitize` as in the other doc string?

Reply:
Fixed.